manifests/40-rbac.yaml

```diff
@@ -69,6 +69,12 @@ rules:
 - apiGroups: ["coordination.k8s.io"]
   resources: ["leases"]
   verbs: ["create","get","update","patch"]
+- apiGroups: ["node.k8s.io"]
```

Comment:
In the past, I was asked by ProdSec to review RBAC.  After the review, I added a few comments here.  Can we also do this for PAO?  Especially here for `runtimeclasses`.  `performance.openshift.io` should be clear to all.

Reply:
Done